cloudify/workflows/workflow_context.py

```diff
@@ -697,6 +701,7 @@ def _build_cloudify_context(self,
             'execution_id': self.execution_id,
             'workflow_id': self.workflow_id,
             'tenant': self.tenant,
+            'host_ip': self.host_ip,
```

Comment:
this is not the right place because this also affects local workflows, where it'll just always be none
instead better add it to the remote-specific handler https://github.com/cloudify-cosmo/cloudify-common/blob/8aa6dacf4f26f84a6937a87f320bd81744bed62c/cloudify/workflows/workflow_context.py#L1413

Reply:
You mean to update this as the following 
```
    @property
    def operation_cloudify_context(self):
        return {'local': False,
                'bypass_maintenance': utils.get_is_bypass_maintenance(),
                'rest_token': utils.get_rest_token(),
                'execution_token': utils.get_execution_token(),
                'host_ip': self.workflow_ctx.host_ip}
```

Reply:
yeah